.github/workflows/objective-c-xcode.yml

```diff
@@ -0,0 +1,34 @@
+name: Xcode - Build and Analyze
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  build:
+    name: Build and analyse default scheme using xcodebuild command
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: download mackernelsdk
+        run: git clone https://github.com/acidanthera/MacKernelSDK
+
+      - run: src=$(/usr/bin/curl -Lfs https://raw.githubusercontent.com/acidanthera/VoodooInput/master/VoodooInput/Scripts/bootstrap.sh) && eval "$src" && mv VoodooInput Dependencies
+      - run: pip3 install --break-system-packages cpplint
+      - run: pip3 install --break-system-packages git+https://github.com/newperson1746/cldoc-fix.git
```

Comment:
This is a security concern and we should change this to pull the cldoc package from the VoodooI2C repo.